.travis.yml

```diff
@@ -14,6 +14,8 @@ branches:
 cache:
     apt: true
     directories:
+        - $HOME/.cargo
+        - $HOME/.rustup
```

Comment:
Is there no gem directory to cache for Ruby?

(I have no special knowledge here, I’m just surprised.)

Reply:
It's actually stored in the ruby directory, which is part of .runtimes which is already cached.

...or it would be if I hadn't put rubygems into deployment mode. This was *meant* to cache `vendor/bundle`, which is [what Travis recommend](https://docs.travis-ci.com/user/caching/#Enabling-multiple-caching-features), but that got lost in some merging shenanigans. Will add.